Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
+  version "1.0.2"
+  sha256 "d77dd150d65836c7b9a12d75d22159484663c11cea5739653bf31c44a3609522"
+
+  head "https://github.com/Yubico/pam-u2f.git"
+
+  depends_on "libu2f-host" => :build
```

Comment:
They can be used separately and independently--my only concern is that it may not be obvious to people that the pam module contains the u2f-host / u2f-server binaries as well. LMK what you'd like.